src/deprecations.jl

```diff
@@ -0,0 +1,2 @@
+# v2
+# @deprecate ErrorDiffusion(filter, offset; color_space=XYZ, kwargs...) ErrorDiffusion(filter, offset, color_space; kwargs...)
```

Comment:
I couldn't get deprecations to work without running into looping function calls due to the `args...; kwargs...` everywhere.

Reply:
@johnnychen94 how do you feel about pushing a patch release with #78 and making this the first commit towards DitherPunk `v3.0.0`?

Reply:
Removed in [a848717](https://github.com/JuliaImages/DitherPunk.jl/pull/73/commits/a848717c7c612078d5075070f807c7662985d368).